<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>981</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Overwiew</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>901</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="temp_png">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>180</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/temperatura.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="hum_png">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>280</y>
         <width>50</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/wilgotnosc.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lux_png">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>380</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/WDS/naswietlenie.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="main_text">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>30</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="temp_line">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>180</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="hum_line">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>280</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lux_line">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>380</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="device_edit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="getdata">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_button">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Charts</string>
     </attribute>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>771</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="temp_png_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Desktop/WDS/temperatura.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_2" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>771</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_3" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>771</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="hum_png_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Desktop/WDS/wilgotnosc.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lux_png_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Desktop/WDS/naswietlenie.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionShow_charts">
   <property name="text">
    <string>Show charts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
